src/mixed-line-bar-chart/data-series.tsx

```diff
@@ -57,11 +59,17 @@ export default function DataSeries<T extends ChartDataTypes>({
     return calculateOffsetMaps(barData);
   }, [visibleSeries, stackedBars]);
 
+  useLayoutEffect(() => {
+    if (!strokeWidth.current && seriesRef.current) {
+      strokeWidth.current = parseInt(getComputedStyle(seriesRef.current).strokeWidth);
```

Comment:
This value will not apply on the current render. Do you see any risks of using stale value here?

Is this over-engineering really necessary? Can we just leave 3-4 pixels to account all possible widths?

Reply:
> This value will not apply on the current render. Do you see any risks of using stale value here?

Right, because changes in refs values will not trigger a re-render. Is that what you mean?

> Is this over-engineering really necessary? Can we just leave 3-4 pixels to account all possible widths?

Main reason why I did it this way is because I assume the clipping has a purpose, i.e, to cut off any lines that stick out of the graph, and I wanted to minimize changes to that behavior, i.e, keep any eventual overflowing graphics to the minimum necessary.

Reply:
> Is that what you mean?

Yes 

> keep any eventual overflowing graphics to the minimum necessary

Still recommend to find a minimal but constant value and put it into the code. Sometimes a hard-coded value performs better than a dynamic detection. If you need an extra example to convince, have a look at these constants: https://github.com/cloudscape-design/components/blob/main/src/internal/components/cartesian-chart/constants.ts

Reply:
Well, and I was mainly doing this because the stroke width is based on a design token (`border-line-chart-width`) but it's actually not exposed, so at the moment there is no way that this can be broken by theming.